src/main/resources/interface-specification.yml

```diff
@@ -4757,17 +4739,17 @@ paths:
               schema:
                 type: integer
               description: Time interval in milliseconds. Allowed requests will be constantly replenished during the interval. At the end of the interval the max allowed requests will be available
-  '/relationships/{relationshipId}':
+  '/users/{userId}':
```

Comment:
@ruggerocastagnola this endpoint is a bit changed now that we have the `userId`.
What does the frontend need here? Just the info of a specific user or the info of the user related to a Tenant (eg name and roles)?

Reply:
What is the difference? The minimal set of info needed about a user in all related endpoints are:
- name
- surname
- id
- role on platform

Right now we also have the tax code but that is not mandatory, in case